packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -1,70 +1,74 @@
-/**
- * These types are exposed when rendering extension points
- */
+import { RawTimeRange, TimeZone } from './time';
 
-export enum PluginExtensionPlacements {
-  DashboardPanelMenu = 'grafana/dashboard/panel/menu',
-}
+// Plugin Extensions types
+// ---------------------------------------
 
 export enum PluginExtensionTypes {
   link = 'link',
-  command = 'command',
 }
 
 export type PluginExtension = {
   type: PluginExtensionTypes;
   title: string;
   description: string;
-  key: number;
```

Comment:
Just leaving a note in case we want to add this back in a future PR?

Reply:
I think we need that one. We can't rely on the index since we would get different extensions back depending on the context.

Reply:
I have reintroduced it as a `id` prop (holding the same hash as before, but now it's a string and not a number - changed it to be a string as I think if we introduce the `id` field in the configs as well, then those will be strings, too, and like this the change can be seamless from the call-site's perspective.)